src/pages/markets/_available-platforms.js

```diff
@@ -54,58 +59,74 @@ const StyledFlex = styled(Flex)`
         padding: 0;
     }
 `
-const AvailablePlatforms = ({ dmt5, dtrader, dbot, smarttrader }) => {
+const AvailablePlatforms = ({
+    dmt5,
+    dtrader,
+    dbot,
+    smarttrader,
+    flex_direction,
+    tablet_direction,
+    tablet_jc,
+}) => {
     return (
-        <PlatformsContainer ai="center">
-            <StyledText>{localize('Available on')}</StyledText>
```

Comment:
please check the `Available on` section again, the result is not the same as the design :pray: